pkg/features/antrea_features.go

```diff
@@ -195,37 +200,38 @@ var (
 	// available throughout Antrea binaries.
 	DefaultAntreaFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-		AntreaPolicy:                {Default: true, PreRelease: featuregate.Beta},
-		AntreaProxy:                 {Default: true, PreRelease: featuregate.GA},
-		BGPPolicy:                   {Default: false, PreRelease: featuregate.Alpha},
-		Egress:                      {Default: true, PreRelease: featuregate.Beta},
-		EndpointSlice:               {Default: true, PreRelease: featuregate.GA},
-		TopologyAwareHints:          {Default: true, PreRelease: featuregate.GA},
-		ServiceTrafficDistribution:  {Default: true, PreRelease: featuregate.GA},
-		CleanupStaleUDPSvcConntrack: {Default: true, PreRelease: featuregate.Beta},
-		Traceflow:                   {Default: true, PreRelease: featuregate.Beta},
-		PacketCapture:               {Default: false, PreRelease: featuregate.Alpha},
-		AntreaIPAM:                  {Default: false, PreRelease: featuregate.Alpha},
-		FlowExporter:                {Default: false, PreRelease: featuregate.Alpha},
-		NetworkPolicyStats:          {Default: true, PreRelease: featuregate.Beta},
-		NodePortLocal:               {Default: true, PreRelease: featuregate.GA},
-		NodeIPAM:                    {Default: true, PreRelease: featuregate.Beta},
-		Multicast:                   {Default: true, PreRelease: featuregate.Beta},
-		Multicluster:                {Default: false, PreRelease: featuregate.Alpha},
-		SecondaryNetwork:            {Default: false, PreRelease: featuregate.Alpha},
-		ServiceExternalIP:           {Default: true, PreRelease: featuregate.Beta},
-		TrafficControl:              {Default: false, PreRelease: featuregate.Alpha},
-		IPsecCertAuth:               {Default: false, PreRelease: featuregate.Alpha},
-		ExternalNode:                {Default: false, PreRelease: featuregate.Alpha},
-		SupportBundleCollection:     {Default: false, PreRelease: featuregate.Alpha},
-		L7NetworkPolicy:             {Default: false, PreRelease: featuregate.Alpha},
-		LoadBalancerModeDSR:         {Default: false, PreRelease: featuregate.Alpha},
-		AdminNetworkPolicy:          {Default: false, PreRelease: featuregate.Alpha},
-		EgressTrafficShaping:        {Default: false, PreRelease: featuregate.Alpha},
-		EgressSeparateSubnet:        {Default: true, PreRelease: featuregate.Beta},
-		NodeNetworkPolicy:           {Default: false, PreRelease: featuregate.Alpha},
-		L7FlowExporter:              {Default: false, PreRelease: featuregate.Alpha},
-		NodeLatencyMonitor:          {Default: false, PreRelease: featuregate.Alpha},
+		AntreaPolicy:                  {Default: true, PreRelease: featuregate.Beta},
+		AntreaProxy:                   {Default: true, PreRelease: featuregate.GA},
+		BGPPolicy:                     {Default: false, PreRelease: featuregate.Alpha},
+		Egress:                        {Default: true, PreRelease: featuregate.Beta},
+		EndpointSlice:                 {Default: true, PreRelease: featuregate.GA},
```

Comment:
I never realized it before, but `FeatureSpec` has a `LockToDefault` field: https://pkg.go.dev/k8s.io/component-base/featuregate#FeatureSpec
I think it is meant to prevents the feature from being disabled, typically after a GA graduation. You get an error (which will crash the antrea-agent) and I think that's what we want here: https://github.com/kubernetes/component-base/blob/2fbd958bc1f70556c227642c181065e1ab195b2c/featuregate/feature_gate.go#L499-L500